### ISSUE TITLE
[BUG]: `magento-sync` improperly defined in `docker-compose.yml` when using `native` sync strategy on Windows w/ WSL2

### ISSUE THREAD

**Issue by damienwebdev** [magento/magento-cloud-docker#214]
### Preconditions
Magento: v2.3.4
Windows 10
WSL2 - Ubuntu 18.04
`"magento/ece-tools": "^2002.1"`

### Steps to reproduce
1. Create an empty Magento Cloud project
2. Install WSL2 on Windows
3. Create auth.json
4. Install the `ece-tools` as a dev dependency
5. Generate the `docker-compose.yml` file
```bash
./vendor/bin/ece-docker build:compose --mode="developer" --sync-engine="native"
```
6. Run the environment:
```
docker-compose up -d
```

### Expected result
1. Given that this is essentially an Ubuntu 18.04 instance, I'd anticipate the yaml file created to look like the native version with relative mounts instead of `magento-sync`.

### Actual result
Get an error
```
ERROR: Volume magento-sync declared as external, but could not be found. Please create the volume manually using `docker volume create --name=magento-sync` and try again.
```

**Comment by damienwebdev** [comment]
cc: @shiftedreality

**Comment by hcmlopes** [comment]
I can verify the bug in WSL2 and my solution based on @damienwebdev recommendations did fix the issue.
I modified the docker-compose.yml as follows
I replace every instance of `- 'magento-sync:/app:nocopy'` with `- '.:/app'` 
then I removed `magento-sync:
    external: true` at the bottom of the file and all worked perfectly fine.

I hope this can assist with making ece-tools more compatible with WSL2 setup.

**Comment by shiftedreality** [comment]
@damienwebdev could you provide a Cloud Docker version and the generated `docker-compose.yml` file?

**Comment by damienwebdev** [comment]
Fixed by #276
